yarn.lock

```diff
@@ -6,9 +6,9 @@ __metadata:
   cacheKey: 10c0
```

Comment:
Why do we have so much `yarn.lock` changes just for an SDK version change and that too in the CC branch? Let us checkout to the yarn.lock from the `ccwidgets` branch, then do a yarn install and check-in that file which should have relatively lesser change set.